commit 1be9a434751b9efeca067efbc4c0050df21082e7
Author: popcornmix <popcornmix@gmail.com>
Date:   Mon Jun 11 16:10:35 2018 +0100

    dwc_otg: Disable fiq by default until it is fixed

diff --git a/drivers/usb/host/dwc_otg/dwc_otg_driver.c b/drivers/usb/host/dwc_otg/dwc_otg_driver.c
index e945900c503c..315f260fff48 100644
--- a/drivers/usb/host/dwc_otg/dwc_otg_driver.c
+++ b/drivers/usb/host/dwc_otg/dwc_otg_driver.c
@@ -241,9 +241,9 @@ static struct dwc_otg_driver_module_params dwc_otg_module_params = {
 };
 
 //Global variable to switch the fiq fix on or off
-bool fiq_enable = 1;
+bool fiq_enable = false;
 // Global variable to enable the split transaction fix
-bool fiq_fsm_enable = true;
+bool fiq_fsm_enable = false;
 //Bulk split-transaction NAK holdoff in microframes
 uint16_t nak_holdoff = 8;
 
diff -uN a/1.txt b/1.txt
--- a/dummy/rpi_1165_1be9a434751b9efeca067efbc4c0050df21082e7.txt	1970-01-01 00:00:00.000000000 +0000
+++ b/dummy/rpi_1165_1be9a434751b9efeca067efbc4c0050df21082e7.txt	2013-12-23 04:07:40.000000000 +0000
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
